reV/supply_curve/points.py

```diff
@@ -43,109 +1043,71 @@ def __init__(self, gid, excl, gen, tm_dset, gen_index, excl_dict=None,
         excl_dict : dict | None
             Dictionary of exclusion LayerMask arugments {layer: {kwarg: value}}
             None if excl input is pre-initialized.
-        resolution : int | None
-            SC resolution, must be input in combination with gid.
+        resolution : int
+            Number of exclusion points per SC point along an axis.
+            This number**2 is the total number of exclusion points per
+            SC point.
+        excl_area : float
+            Area of an exclusion cell (square km).
         exclusion_shape : tuple
-            Shape of the exclusions extent (rows, cols). Inputing this will
-            speed things up considerably.
+            Shape of the full exclusions extent (rows, cols). Inputing this
+            will speed things up considerably.
         offshore_flags : np.ndarray | None
             Array of offshore boolean flags if available from wind generation
             data. None if offshore flag is not available.
         close : bool
             Flag to close object file handlers on exit.
         """
 
-        self._gid = gid
-        self._close = close
-        self._centroid = None
-        self._excl_data = None
-        self._excl_data_flat = None
-        self._excl_dict = excl_dict
-
-        # filepaths
-        self._excl_fpath = None
-        self._gen_fpath = None
+        super().__init__(gid, excl, gen, tm_dset,
+                         excl_dict=excl_dict,
+                         resolution=resolution,
+                         excl_area=excl_area,
+                         exclusion_shape=exclusion_shape,
+                         close=close,
+                         parse_tm=False)
 
-        # handler objects
-        self._exclusions = None
-        self._gen = None
+        self._res_gid_set = None
+        self._gen_gid_set = None
 
-        # Parse inputs
-        self._parse_files(excl, gen)
-        self._rows, self._cols = self._parse_slices(
-            gid, resolution, exclusion_shape=exclusion_shape)
+        self._gen_fpath, self._gen = self._h5_fpath, self._h5
         self._gen_gids, self._res_gids = self._parse_techmap(tm_dset,
                                                              gen_index)
         self._remove_offshore(self._gen_gids, self._res_gids,
                               offshore_flags=offshore_flags)
 
-    def _parse_files(self, excl, gen):
-        """Parse gen + excl filepath input or handler object and set to attrs.
+    @staticmethod
+    def _parse_h5_file(gen):
```

Comment:
This method is redundant to the static method in `AggregationSupplyCurvePoint`

Reply:
Not quite, in AggregationSupplyCurvePoint we initialize a Resource handler, while in GenSCPoint we initialize an Outputs handler.  I honestly not sure if we need the Outputs handler so we could remove this...

Reply:
Yeah I dont think we need the outputs handler but its not a big deal either way